android/src/main/java/io/invertase/googlemobileads/common/ReactNativeAdView.java

```diff
@@ -2,6 +2,7 @@
 
 import android.content.Context;
 import android.widget.FrameLayout;
+import com.facebook.react.bridge.ReadableArray;
```

Comment:
this import seems to have been added but there are not other diff `+` lines that reference it ?